src/js/components/NewsFeedPost.jsx

```diff
@@ -25,98 +22,58 @@ export default class NewsFeedPost extends React.Component {
     console.log(NewsFeedStoreStore.error);
   }
 
-  togglePictureDisplay() {
-    this.setState({
-      showPicture: !this.state.showPicture
-    });
-  }
-
   changeLike() {
-    // TODO: How are we doing this on the backend?
-    if (this.state.liked) {
-      // When this works, remove liked attribute
+    if (this.state.current_user_likes) {
       NewsFeedStore.decrementLike();
       this.setState({
-        liked: 0,
-        likeStyle: {color: 'black', fontSize: "40px", marginLeft: "50px"},
+        current_user_likes: false,
       });
     } else {
       NewsFeedStore.incrementLike();
       this.setState({
-        liked: 1,
-        likeStyle: {color: 'red', fontSize: "40px", marginLeft: "50px"},
-      })
+        current_user_likes: true,
+      });
     }
 
   }
 
   render() {
 
-    const { id, posterid, text, like_count, media_link, timestamp } = this.props;
-
-    // TODO: This is a dirty hack job until the API has a solution
-    let likeCount = like_count + this.state.liked;
-
-
-
-    var width100 = {
-      margin: "30px -10% 15px 20%",
-    }
-
-    var font30 = {
-      fontSize: "26px"
-    }
-
-    var font40 = {
-      fontSize: "32px"
-    }
+    let { current_user_likes, like_count, poster_icon,
+          poster_id, poster_name, text, media_link } = this.state;
 
-    var font24 = {
-      fontSize: "20px",
+    const heartColor = {
+      color: current_user_likes ? "red" : "inherit",
     }
 
-    var fleft = {
-      float: "left"
-    }
-
-    var fcenter = {
-      float: "center"
-    }
-
-
-    let renderPicture;
-    if (this.props.media_link === undefined || this.props.media_link === "") {
-      renderPicture = (
-        <span style={font24}>NO MEDIA LINK</span>
-      );
-    } else if (!this.state.showPicture) {
-      renderPicture = (
-        <span class="picture-text" style={font30} onClick={this.togglePictureDisplay}>Show Photo...</span>
-      );
-    } else {
-      renderPicture = (
-        <span class="picture" onClick={this.togglePictureDisplay}>
-          <img src={media_link}/>
+    let imageText = null;
+    if (media_link && media_link !== "") {
+      imageText = (
+        <span className="show-image link clickable no_selection">
+          Show Image
         </span>
       );
     }
 
-
-
-
     return (
-      <div class="newsfeed-post" style={width100}>
-        <div class="test-post">
-          <span class="username" style={font40}>{posterid} - </span>
-          <span class="text" style={font30}>{text} </span>
+      <div className="newsfeed-post">
+        <div className="user-photo-container inline">
+          <img src={poster_icon} className="user-photo"/>
+        </div>
+        <div className="post-text inline">
+          <p className="username">{poster_name}</p>
+          <p className="text">{text}</p>
+          <div className="social-bar">
+            <FaHeart className="heart-icon clickable" onClick={this.changeLike} style={heartColor} size={20}/>
```

Comment:
@schneider-clarisse should we just have a class name that we toggle, or is inline more correct? Not sure what the react norms for this are.